### PR TITLE
home page fixes and suggestion

### PR DESCRIPTION
**Pull request by olivercodes**
1) Fix right side table of contents to include all sections on the page
2) Add optional suggestion of KEP list

before
image

after
image

### CODE REVIEW

File: site-src/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Introduction
+## Introduction
```
Comment (ryanzhang-oss) [inline site-src/index.md:1]: I wonder why move everything one level down?

Comment (olivercodes) [reply site-src/index.md:1]: The Table of Contents on the right side of the page was ignoring any singular `#` sections. I thought that was a bit confusing for new readers to the page. Moving each one a level down fixes this

image

Comment (skitt) [reply site-src/index.md:1]: I think it’s better to fix the configuration (#31) instead of manually shifting everything down a level.

Comment (olivercodes) [reply site-src/index.md:1]: Yup, that's much better. Closing this

Comment (skitt) [reply site-src/index.md:1]: Could you keep the added KEP links?

Comment (olivercodes) [reply site-src/index.md:1]: Sure! I'll reopen with that

Comment (olivercodes) [reply site-src/index.md:1]: https://github.com/kubernetes-sigs/sig-multicluster-site/pull/35

**Pull request closed without merging**